src/ui/public/partials/saved_object_finder.html

```diff
@@ -1,38 +1,56 @@
-<form role="form" class="form-inline">
-  <div class="container-fluid">
-    <div class="row">
-      <div class="input-group form-group finder-form col-md-7">
-        <span class="input-group-addon savedObjectFinderSearchIcon">
-          <span class="fa fa-search"></span>
-        </span>
+<form
+  role="form"
+  class="kuiVerticalRhythm"
+>
+  <div class="kuiBar">
+    <div class="kuiBarSection">
+      <div class="kuiSearchInput fullWidth">
+        <div class="kuiSearchInput__icon kuiIcon fa-search"></div>
         <input
+          class="kuiSearchInput__input"
           input-focus
           ng-model="filter"
-          ng-attr-placeholder="{{finder.properties.nouns | label }} Filter..."
+          ng-attr-placeholder="{{ finder.properties.nouns | label }} Filter..."
           ng-keydown="finder.filterKeyDown($event)"
-          class="form-control"
           name="filter"
           type="text"
           autocomplete="off"
-        />
-      </div>
-      <div class="finder-hit-count col-md-2">
-        <span>{{finder.hitCount}} of {{finder.hitCount}}</span>
+        >
       </div>
-      <div class="finder-manage-object col-md-2" ng-if="onAddNew">
-        <a class="small" ng-click="onAddNew()" data-test-subj="addNewSavedObjectLink">
-          Add new {{finder.properties.noun}}
-        </a>
-      </div>
-      <div class="finder-manage-object col-md-2" ng-if="!useLocalManagement">
-        <a class="small" ng-click="finder.manageObjects(finder.properties.name)">
-          Manage {{finder.properties.nouns}}
-        </a>
+    </div>
+
+    <div class="kuiBarSection">
+      <p class="kuiText kuiSubduedText">
+        {{ finder.hitCount }} of {{ finder.hitCount }}
```

Comment:
I know you didn't change this at all, but I don't know how helpful an `x of y` is if x === y all the time.

I'm guessing the first part is supposed to be the count on the first page?  Instead it'll just show all the items, like `100 of 100` even though only 20 many be showing on the current page.

Since this is existing, im fine if you'd prefer I file a separate issue.  Just mentioning while you are in here, might want to fix it.